airflow/providers/vertica/hooks/vertica.py

```diff
@@ -45,6 +45,51 @@ def get_conn(self) -> connect:
             conn_config["port"] = 5433
         else:
             conn_config["port"] = int(conn.port)
+            
+        if conn.extra_dejson.get("connection_load_balance", False):
```

Comment:
Can we fetch `extra` dict by calling `conn.extra_dejson` once rather than calling it repeatedly? It makes it look very wordy

Reply:
Yes, I based my code on what it was done for the mysql hook, I didn't realize that extra_dejson was property that load json every time you call it.

Reply:
It's not that big of a performance concern since it would happen only while adding the connection (which is one time process). But the code looks very wordy.